app/editor/src/features/admin/topics/TopicList.tsx

```diff
@@ -36,11 +36,12 @@ export const TopicList: React.FC = () => {
   const [loading, setLoading] = React.useState(false);
   const [topics, setTopics] = React.useState<ITopicModel[]>([]);
   const [items, setItems] = React.useState<ITopicModel[]>([]);
+  const [topicFilter, setTopicFilter] = React.useState('');
 
   const topicTypeOptions = getEnumStringOptions(TopicTypeName);
 
   React.useEffect(() => {
-    if (!items.length && !loading) {
+    if (!items.length && !loading && !topicFilter) {
```

Comment:
Can you explain this change?  It will find all topics only if there is no topic filter?

Reply:
Without this there could be an issue when there is no data matched to the topic filter, in that case it will show all records instead of an expected empty list.